src/pyodide-py/pyodide/console.py

```diff
@@ -119,6 +123,7 @@ def __init__(
         stdout_callback: Optional[Callable[[str], None]] = None,
         stderr_callback: Optional[Callable[[str], None]] = None,
         persistent_stream_redirection: bool = False,
+        completion=True,
```

Comment:
Maybe let's make it False by default since jedi is not guaranteed to be installed.

Reply:
This will not fail actually.

Reply:
I know but it will return an empty completion list, which is also unexpected..

Reply:
OK, I've seen your comment below.